site/content/en/docs/manual/advanced/annotation-with-brushing-tools.md

```diff
@@ -0,0 +1,121 @@
+
+Do the following:
+
+1. From the [controls sidebar](/docs/manual/basics/controls-sidebar/), select **Brush** ![Brush icon](/images/brushing_tools_icon.png).
+2. In the **Draw new mask** menu, select label for your mask, and click **Draw a mask**. <br>The **Brush**![Brush](/images/brushing_tools_icon.png) tool will be selected by default.
+
+   ![BT context menu](/images/brushing_tools_context_menu.png)
+
+3. With the brush, draw a mask on the object you want to label. <br>To correct selection, use **Eraser** ![Eraser](/images/brushing_tools_erase.png)
+
+   ![Brushing](/images/brushing_tools.gif)
+
+4. After you applied the mask, on the top menu bar click **Save mask** ![Tick icon](/images/tick_icon.png) to finish the process (or **N** on the keyboard).
+5. Added label will appear on the [objects sidebar](/docs/manual/basics/objects-sidebar/).
+
+To add the next label, repeat steps 1 to 5. All added labels will be visible on the image and the  [objects sidebar](/docs/manual/basics/objects-sidebar/).
+
+To save the project with all added labels, on the top menu click **Save** ![Save](/images/brushing_tools_save.png).
+
+
+
+## Annotation with polygon-to-mask
+
+Prerequisites: [Create task](/docs/manual/basics/creating_an_annotation_task/), upload the image, add labels, and open the workspace.
+
+Do the following:
+
+1. From the From the [controls sidebar](/docs/manual/basics/controls-sidebar/), select **Brush** ![Brush icon](/images/brushing_tools_icon.png).
+2. In the **Draw new mask** menu, select label for your mask, and click **Draw a mask**.
+
+   ![BT context menu](/images/brushing_tools_context_menu.png)
+
+3. In the brushing tool menu, select **Polygon** ![Add poly](/images/brushing_tools_add_poly.png).
+4. With the **Polygon**![Add poly](/images/brushing_tools_add_poly.png) tool, draw a mask for the object you want to label. <br>To correct selection, use **Remove polygon selection** ![Remove poly](/images/brushing_tools_remove_poly.png).
+5. Use **Save mask** ![Tick icon](/images/tick_icon.png) (or **N** on the keyboard) to switch between add/remove polygon tools:
+
+   ![Brushing](/images/brushing_tools_polygon.gif)
+
+6. After you added the polygon selection, on the top menu bar click **Save mask** ![Tick icon](/images/tick_icon.png) to finish the process (or **N** on the keyboard).
+7. Added label will appear on the [objects sidebar](/docs/manual/basics/objects-sidebar/).
+
+To add the next label, repeat steps 1 to 5. All added labels will be visible on the image and the  [objects sidebar](/docs/manual/basics/objects-sidebar/).
+
+To save the project with all added labels, on the top menu click **Save** ![Save](/images/brushing_tools_save.png).
+
+
+## Remove underlying pixels
+
+Use **Remove underlying pixels** tool when you want to draw a new mask on top of the other masks and simultaneously delete the pixels of other masks that are located at the same positions.
```

Comment:
```suggestion
Use **Remove underlying pixels** tool when you want to delete the pixels of other masks that are located at the same positions. It is highly useful feature to avoid meticulous drawing edge twice between two different objects
```